__manifest__.py

```diff
@@ -1,60 +1,54 @@
 # -*- coding: utf-8 -*-
 {
-    'name': "gse_custo",
-
-    'summary': """
+    "name": "gse_custo",
+    "summary": """
         Customizations pour Goshop Energy""",
```

Comment:
@iambenkis 
Ave you seen any official Odoo module with the double quote instead of the single quote?